<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonIngresar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(128, 128, 128, 255), stop:1 rgba(255, 255, 255, 255));
font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ingresar </string>
    </property>
   </widget>
   <widget class="QWidget" name="W_Inicio" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(128, 128, 128, 255), stop:1 rgba(255, 255, 255, 255));
font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Inicio</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>0</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="recursos.qrc">:/images/imagenes/Imagen1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBoxTipoUsuario">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCorreo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPass">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsuario">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="QL_TUsuario">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Tipo de Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="QL_Correo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>275</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Correo Electrónico:</string>
    </property>
   </widget>
   <widget class="QLabel" name="QL_Pass">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLabel" name="QL_Usuario">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-90</y>
      <width>881</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/imagenes/imgfondo_1.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButtonIngresar</zorder>
   <zorder>W_Inicio</zorder>
   <zorder>comboBoxTipoUsuario</zorder>
   <zorder>lineEditCorreo</zorder>
   <zorder>lineEditPass</zorder>
   <zorder>lineEditUsuario</zorder>
   <zorder>QL_TUsuario</zorder>
   <zorder>QL_Correo</zorder>
   <zorder>QL_Pass</zorder>
   <zorder>QL_Usuario</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
